bindata/v4.1.0/kube-apiserver/pod.yaml

```diff
@@ -43,6 +43,15 @@ spec:
     command: ["/bin/bash", "-ec"]
     args:
         - |
+          echo -n "Acquiring exclusive lock /var/log/kube-apiserver/.lock"
+          exec {lock_fd}>/var/log/kube-apiserver/.lock || exit 1
+          flock -n "${lock_fd}" || {
```

Comment:
this is for acquiring the lock. I have checked by default exclusive is applied and `-n` here means locking is a non-blocking operation.

Reply:
where the lock is released?

Reply:
automatically by leaving the shell.